File: noti-send-service/src/main/java/com/ezbuy/notisendservice/client/properties/AuthProperties.java
package com.ezbuy.notisendservice.client.properties;

import com.reactify.client.properties.WebClientProperties;
import lombok.AllArgsConstructor;
import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Component("authProperties")
@ConfigurationProperties(prefix = "client.auth", ignoreInvalidFields = true)
@Data
@AllArgsConstructor
public class AuthProperties extends WebClientProperties { }

File: notification-send/src/main/kotlin/com/ezbuy/notificationsend/client/properties/AuthProperties.kt
package com.ezbuy.notificationsend.client.properties

import com.reactify.client.properties.WebClientProperties
import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@Component("authProperties")
@ConfigurationProperties(prefix = "client.auth", ignoreInvalidFields = true)
class AuthProperties : WebClientProperties()